tests/components/modbus/test_binary_sensor.py

```diff
@@ -19,9 +19,10 @@
 )
 from homeassistant.core import State
 
-from .conftest import ReadResult, base_config_test, base_test, prepare_service_update
+from .conftest import ReadResult, base_test, prepare_service_update
 
-from tests.common import mock_restore_cache
+sensor_name = "test_binary_sensor"
+entity_id = f"{SENSOR_DOMAIN}.{sensor_name}"
```

Comment:
These are constants so we should use all uppercase in the variable name, eg `ENTITY_ID`.

Reply:
Solved in #52337